src/object.d

```diff
@@ -49,6 +49,14 @@ public import rt.array.capacity: reserve;
 /// See $(REF assumeSafeAppend, rt,array,capacity)
 public import rt.array.capacity: assumeSafeAppend;
 
+/// See $(REF capacity, rt,array,capacity)
+public import rt.array.capacity: capacity;
+
+/// See $(REF _d_arrayappendT, rt,array,append) and $(REF _d_arrayappendTTrace, rt,array,append)
```

Comment:
You actually don't need to document these, because they are not called directly by the user.  `capacity.d` has functions that are called directly by the user, so it is an exception.  I'd like to add a `version(Core_Doc)` block to filter out the runtime hooks so they don't appear in the documentation.

Reply:
Okey, thanks.

It think it is better to do the `version(Core_Doc)` in a separate PR.